<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesColor</class>
 <widget class="QWidget" name="PreferencesColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,2">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Background</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="UJ::ColorWell" name="colorBackground" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,0,50,50,0">
      <item row="6" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Magenta</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>White</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Red</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Yellow</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Cyan</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Black</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="UJ::ColorWell" name="colorBlack" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="UJ::ColorWell" name="colorBlackBold" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="UJ::ColorWell" name="colorRed" native="true"/>
      </item>
      <item row="2" column="3">
       <widget class="UJ::ColorWell" name="colorRedBold" native="true"/>
      </item>
      <item row="3" column="2">
       <widget class="UJ::ColorWell" name="colorGreen" native="true"/>
      </item>
      <item row="4" column="2">
       <widget class="UJ::ColorWell" name="colorYellow" native="true"/>
      </item>
      <item row="3" column="3">
       <widget class="UJ::ColorWell" name="colorGreenBold" native="true"/>
      </item>
      <item row="4" column="3">
       <widget class="UJ::ColorWell" name="colorYellowBold" native="true"/>
      </item>
      <item row="5" column="2">
       <widget class="UJ::ColorWell" name="colorBlue" native="true"/>
      </item>
      <item row="5" column="3">
       <widget class="UJ::ColorWell" name="colorBlueBold" native="true"/>
      </item>
      <item row="6" column="2">
       <widget class="UJ::ColorWell" name="colorMagenta" native="true"/>
      </item>
      <item row="6" column="3">
       <widget class="UJ::ColorWell" name="colorMagentaBold" native="true"/>
      </item>
      <item row="7" column="2">
       <widget class="UJ::ColorWell" name="colorCyan" native="true"/>
      </item>
      <item row="7" column="3">
       <widget class="UJ::ColorWell" name="colorCyanBold" native="true"/>
      </item>
      <item row="8" column="2">
       <widget class="UJ::ColorWell" name="colorWhite" native="true"/>
      </item>
      <item row="8" column="3">
       <widget class="UJ::ColorWell" name="colorWhiteBold" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UJ::ColorWell</class>
   <extends>QWidget</extends>
   <header>ColorWell.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
